source/manual/govuk-notify.html.md

```diff
@@ -24,7 +24,9 @@ for each environment):
 
   This service is used by Email Alert API only. It's used to send public-facing
   email updates about pieces of content on GOV.UK. It's our biggest sender of
-  emails and regularly exceeds one million emails per day.
+  emails and averages around 4 million emails per day.
+
+  This [dashboard][Email Alert API Product dashboard] shows usage over time.
```

Comment:
```suggestion
  The [Email Alert API Product dashboard] shows usage over time.
```

I'd suggest this wording because then we see the name of the dashboard in the sentence.